doc/user/content/overview/api-components.md

```diff
@@ -8,8 +8,7 @@ menu:
 ---
 
 Materialize is a streaming database with a SQL API. However, despite the fact
-that Materialize uses SQL idioms and can process data from databases, it
-actually has very little in common with "databases" as most users think of them.
+that Materialize uses SQL idioms and can process data from databases, it actually has very little in common with "databases" as most users think of them.
```

Comment:
Just a heads up that these line breaks are intentional and should be maintained; all text should be wrapped at some character count. Most editors have some kind of plugin that will do this for you.